src/components/table/TableCellRenderer.tsx

```diff
@@ -54,7 +57,7 @@ export function CellDateRenderer({
   const classes = useStyles();
 
   return (
-    <TableCell id={id} align='left' className={classes.date + ' ' + className}>
+    <TableCell id={id} align='left' className={classes.date + ' ' + classes.default + ' ' + className}>
```

Comment:
another way.. if you think it might be better:
className={`${classes.date}  ${classes.default} ${className}`}

Reply:
Oh that's cleaner. I'll make the change 👍

Reply:
Thanks 🎉